{"name": "test_demo", "status": "failed", "statusDetails": {"message": "AssertionError: False is not true", "trace": "self = <unit.test_demo.TestDemo testMethod=test_demo>\n\n    def test_demo(self):\n>       self.assertTrue(False)\nE       AssertionError: False is not true\n\nunit\\test_demo.py:25: AssertionError"}, "attachments": [{"name": "stdout", "source": "16886179-7213-45f9-80b1-678980d8f4ad-attachment.txt", "type": "text/plain"}], "start": 1582965707935, "stop": 1582965707936, "uuid": "aa6a84b3-523c-4958-8afc-dfd4d67f3fef", "historyId": "30c1e5a57ae333ffbe92bfa2fcc203b6", "testCaseId": "609a731e35f5ad87b7ed57d5b94a1d0f", "fullName": "unit.test_demo.TestDemo#test_demo", "labels": [{"name": "parentSuite", "value": "unit"}, {"name": "suite", "value": "test_demo"}, {"name": "subSuite", "value": "TestDemo"}, {"name": "host", "value": "LAPTOP-GLFLRPL6"}, {"name": "thread", "value": "20968-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "unit.test_demo"}]}